Practice/Practice/Controllers/EmployeeController.cs

```diff
@@ -79,7 +80,7 @@ public async Task<ActionResult> Create(FormCollection formCollection)
 
             return Json(new { status = "success", message = "Employee has been successfully created." });
         }
-
+        [Authorize]
```

Comment:
Jak narzucasz właściwość na kontroler to wszystkie akcje w nim już tę właściwość mają - w takim razie to jest zbędne